core/controllers/oppia_root.py

```diff
@@ -51,4 +51,6 @@ class OppiaLightweightRootPage(base.BaseHandler):
     @acl_decorators.open_access # type: ignore[misc]
     def get(self, **kwargs: Dict[str, str]) -> None:
         """Handles GET requests."""
-        self.render_template('lightweight-oppia-root.mainpage.html') # type: ignore[no-untyped-call]
+        if self.request.cookies.get('dir') == 'rtl':
+            self.render_template('lightweight-oppia-root.mainpage.html') # type: ignore[no-untyped-call]
```

Comment:
```suggestion
            self.render_template('lightweight-oppia-root.mainpage.html') # type: ignore[no-untyped-call]
            return
```

Not sure if it is needed.

Reply:
Fixed by Vojta himself